tests/unit/helpers/eventsRepetition.test.js

```diff
@@ -444,14 +507,36 @@ describe('createRepetitionsByWeek', () => {
     );
   });
 
-  it('should create repetition every two weeks', async () => {
+  it('should create repetition by week from first event to current time plus 90 days', async () => {
+    const sector = new ObjectID();
+    const event = { startDate: '2019-01-10T09:00:00.000Z', endDate: '2019-01-10T11:00:00Z', customer: new ObjectID() };
+    const range = Array.from(
+      moment().range(moment('2019-01-17T09:00:00.000Z'), moment('2019-04-20T10:00:00.000Z')).by('weeks', { step: 1 })
+    );
+
+    dayDiff.returns(10);
+
+    await EventsRepetitionHelper.createRepetitionsByWeek(event, sector, 1);
+
+    sinon.assert.calledOnceWithExactly(
+      createRepeatedEvents,
+      event,
+      sinon.match(calledRange => JSON.stringify(calledRange) === JSON.stringify(range)),
+      sector,
+      false
+    );
+  });
+
+  it('should create repetition every two weeks from first event to 90 days after', async () => {
```

Comment:
should create repetition every two weeks from today

Reply:
done